<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>153</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>153</width>
     <height>153</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="background.qrc">:/background/background.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="mic_listening_gif">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>153</width>
     <height>153</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="mic.qrc">:/mic/mic_listening.gif</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
  <include location="mic.qrc"/>
 </resources>
 <connections/>
</ui>
